config/console/event-dispatcher.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Composer\Config\Builder;
+use Yiisoft\EventDispatcher\Provider\ListenerCollection;
+use Yiisoft\EventDispatcher\Support\ListenerCollectionFactory;
+
+return [
+    ListenerCollection::class => static fn (ListenerCollectionFactory $factory) => $factory->create(require Builder::path('events-console')),
```

Comment:
Already defined in https://github.com/yiisoft/event-dispatcher/pull/55/files#diff-772dc4129698ed51c6d471cbdb0a770a24702012d18b5ee389e2436800cb5ce1R10